apps/api/src/chapter/adminChapter.service.ts

```diff
@@ -213,7 +213,27 @@ export class AdminChapterService {
   async updateFreemiumStatus(chapterId: UUIDType, isFreemium: boolean) {
     return await this.adminChapterRepository.updateFreemiumStatus(chapterId, isFreemium);
   }
+  async updateChapterDisplayOrder(
+    chapterArray: { chapterId: UUIDType; displayOrder: number }[],
+  ): Promise<void> {
+    if (!chapterArray || chapterArray.length === 0) {
+      throw new BadRequestException("No chapters provided.");
+    }
 
```

Comment:
It would be good to check if all chapters belong to one course and if they are all of course